app/representers/extra_fields_representer.rb

```diff
@@ -1,7 +1,6 @@
 module ExtraFieldsRepresenter
   def representable_attrs
-    # Representable::Config#clone does not work, just create new one and inherit it
-    attrs = Representable::Config.new.inherit!(super)
+    attrs = super
```

Comment:
Not sure whether there is any issue with just calling this. I guess the problem is that it changes the original `representable_attrs`, but I don't know whether that can be problematic.